modules/database/src/ioc/misc/iocInit.c

```diff
@@ -129,6 +130,16 @@ static int iocBuild_1(void)
         errlogPrintf("iocBuild: " ERL_ERROR " Aborting, bad database definition (DBD)!\n");
         return -1;
     }
+
+    {
+        /* fall back to "old" decimal-only conversion if EPICS_DB_CONVERT_DECIMAL_ONLY
+           is set and does not look like meaning no/false/0 */
+        const char* dec_only = getenv("EPICS_DB_CONVERT_DECIMAL_ONLY");
+        if (dec_only && *dec_only && !strchr("nNfF0", *dec_only)) {
```

Comment:
Core Group: Accept `YES` case-independent only, otherwise leave as 0;

Reply:
Changed to: `if (dec_only && epicsStrCaseCmp(dec_only, "YES") == 0)`